cms/templates/js/content-group-edit.underscore

```diff
@@ -0,0 +1,19 @@
+<form class="group-configuration-edit-form">
+    <div class="wrapper-form">
+    <% if (error && error.message) { %>
+        <div class="group-configuration-edit-error message message-status message-status error is-shown" name="group-configuration-edit-error">
+          <%= gettext(error.message) %>
+        </div>
+    <% } %>
+        <fieldset class="group-configuration-fields">
+            <div class="input-wrap field text required add-group-configuration-name <% if(error && error.attributes && error.attributes.name) { print('error'); } %>">
+                <label for="group-cohort-name-<%= uniqueId %>"><%= gettext("Content Group Name") %></label>
```

Comment:
This "label for" is wrong and needs to be fixed. The string referenced by "for" should be the ID on the input, not the name. If you open this file in Pycharm, you will see it highlights the error.

The trick will be creating a unique ID. We do this other places in the code, so I will try to find a reference.